<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BehaviorWidget</class>
 <widget class="QWidget" name="BehaviorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Behavior</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QFrame" name="frmBehavior1">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chkBRecheckDependants">
        <property name="text">
         <string>Recheck dependant test items when master test status has been changed</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lblBConsiderStatus">
          <property name="text">
           <string>Consider status of the master test obsolete after</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBMastersLifetime">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="maximum">
           <number>3600</number>
          </property>
          <property name="value">
           <number>60</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblBSeconds">
          <property name="font">
           <font>
            <family>MS Sans Serif</family>
           </font>
          </property>
          <property name="text">
           <string>seconds</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lblBDontStart">
          <property name="text">
           <string>Don't start more than </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBMaxThreads">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
          <property name="value">
           <number>32</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblBTestsPerSeconts">
          <property name="text">
           <string>tests per second</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmBehavior2">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="lblBResetOnEdinig">
        <property name="text">
         <string>Reset &quot;Recurrences&quot; and refresh test status after editing test properties</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbBResetOnEditing">
        <item>
         <property name="text">
          <string>Reset &quot;Recurrences&quot; without confirmation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ask confirmation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Don't reset</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBResetOnPause">
        <property name="text">
         <string>Reset &quot;Recurrences&quot; when test gets paused</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBRequestForComment">
        <property name="text">
         <string>Request for comment when test gets disabled</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBResetOnDisabled">
        <property name="text">
         <string>Reset &quot;Recurrences&quot; when test gets disabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
